src/random-seed/random-seed.c

```diff
@@ -278,6 +321,7 @@ static int run(int argc, char *argv[]) {
                 r = loop_write(seed_fd, info->buf, (size_t) k, false);
                 if (r < 0)
                         return log_error_errno(r, "Failed to write new random seed file: %m");
+                write_entropy_xattr(seed_fd, MIN(e1, e2));
```

Comment:
what i don't like here though: you have an xattr here you called "entropy_count" and a cmdline arg/env var called "entropy_credit". To me it's not clear at all what the difference is or the unit. The latter takes a permille and the former stores an absolute number of bits. I think that's quite confusing, and they should carry different names, and explicit names. Maybe call the xattr "entropy_credit_bits". And internally call the cmdline arg `arg_entropy_share_permille` (or maybe "ratio" instead of "share"? or "fraction"? or "quota"), but I figure for command line args it's not customary to contain units in the name, hence maybe drop it there, i.e. simply use --entropy-share=, but take permille.